tests/pytest_tests/unit_tests/test_artifacts/test_storage.py

```diff
@@ -25,22 +24,22 @@
 
 
 def test_opener_rejects_append_mode(artifacts_cache):
-    _, _, opener = artifacts_cache.check_md5_obj_path(base64.b64encode(b"abcdef"), 10)
+    _, _, opener = artifacts_cache.check_md5_obj_path(example_digest, 7)
 
     with pytest.raises(ValueError):
         with opener("a"):
             pass
 
     # make sure that the ValueError goes away if we use a valid mode
-    with opener("w"):
-        pass
+    with opener("w") as f:
+        f.write("example")
 
 
 def test_check_md5_obj_path(artifacts_cache):
-    md5 = base64.b64encode(b"abcdef")
-    path, exists, opener = artifacts_cache.check_md5_obj_path(md5, 10)
+    md5 = md5_string("hi")
+    path, exists, opener = artifacts_cache.check_md5_obj_path(md5, 6)
```

Comment:
Is the `size` parameter here supposed to be 2 or did I misunderstand how it is set?

Reply:
Nope, mistake on my part.
```suggestion
    path, exists, opener = artifacts_cache.check_md5_obj_path(md5, 2)
```